src/net.cpp

```diff
@@ -2193,7 +2193,7 @@ void CNode::RecordBytesSent(uint64_t bytes)
 void CNode::SetMaxOutboundTarget(uint64_t limit)
 {
     LOCK(cs_totalBytesSent);
-    uint64_t recommendedMinimum = (nMaxOutboundTimeframe / 600) * MAX_BLOCK_SERIALIZED_SIZE;
+    uint64_t recommendedMinimum = (nMaxOutboundTimeframe / 600) * MAX_BLOCK_BASE_SIZE;
```

Comment:
Though, `MAX_BLOCK_SERIALIZED_SIZE` is your cost "on the wire". This will be true after segwit is active.

I'd rather keep this line as is and adjust the documentation.